README.md

```diff
@@ -228,11 +231,6 @@ Legend:
 
 ## Known Issues
 
-- The SDK currently depends on the hosted version on
-  [sentry.io](https://sentry.io). The latest on-premise version of Sentry (10.0)
-  does not provide server-side support for events sent by `sentry-native`.
-  Full support for `sentry-native` will be made available to all on-premise
-  customers with the next release.
```

Comment:
Has this been made available now? Which release here https://github.com/getsentry/onpremise/releases does this correspond to?

Reply:
You can use the SDK with v20.6.0 or later (of course excluding v10.0.1).